### PR TITLE
Issue 1081 new

### PR DESCRIPTION
**Pull request by richelleyy**

### CODE REVIEW

**Comment by clee1060** [comment]
Did you mean to merge these commits into kitschy.kolache? Right now it's merging into master

File: docs/tutorial/framework-t.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```
Comment (helenemccarron) [inline docs/tutorial/framework-t.html]: you included the tutorial files again

File: styles/books/generic/book.scss [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
-@import 'framework/framework';
-@import 'templates/generic/design';
-@import 'templates/generic/generic';
+@import '../../framework/framework';
+@import '../../templates/generic/design';
+@import '../../templates/generic/generic';
 
 //❓should we use at the template level? How many books use the same spaces?
-@include use('commonStyles', "commonBookStuff");
+@include use('globalStyles', "globalBookStuff");
```
Comment (helenemccarron) [inline styles/books/generic/book.scss]: this looks like an old version of this file, it probably should be reverted

File: styles/designs/carnival/_eoc-components.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
+//header
```
Comment (helenemccarron) [inline styles/designs/carnival/_eoc-components.scss]: this looks like the changes in PR https://github.com/openstax/cnx-recipes/pull/1131. While this is sometimes necessary, I wouldn't add those changes here, especially since they are still under review in the original PR.

File: styles/designs/carnival/_eoc-spaces.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
+@import '_eoc-components';
+
```
Comment (helenemccarron) [inline styles/designs/carnival/_eoc-spaces.scss]: same as above

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+$font_fam:(
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_figureimage-components.scss]: should this be in typography instead? Is it even required, you should be able to use global typography values

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    _subselector: "",
+     _properties:(
+         font-family: enum('ValueSet:::NOT_NULL'),
+         font-weight: enum('ValueSet:::NULLABLE'),
+         color: enum('ValueSet:::NULLABLE'),
+         font-size: enum('ValueSet:::NOT_NULL'),
+         )
+);
+
+$pic_padding:(
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_figureimage-components.scss]: I would much rather use components to match html tags (such as `figure`) instead of more conceptualize concepts such as "padding". You may want to re-use a figure shape, not a set of padding instructions.

File: styles/designs/common/pdf/_common-figure-components.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        margin-right: enum('ValueSet:::NOT_NULL'),//auto
+    )
+);
+
+//images w/o caption/figure number
+$leftalign:(
+    _name: "leftalignImage",
+    _subselector: " > [data-alt = ""] > img",
+     _properties:(
+         float:enum('ValueSet:::NOT_NULL'),//left
```
Comment (helenemccarron) [inline styles/designs/common/pdf/_common-figure-components.scss]: You may want to test this in the PDF, usually, floats do not work well in PrinceXML

File: styles/designs/generic/_splash-components.scss [hunk 8]
```diff
@@ -0,0 +1,32 @@
+//copying styles from ./styles-old/themes/theme3/style/splash-image. NOTE: creating the different pieces for the framework without knowing what CSS structure I will need is virtually impossible. I opened the file and I am refering to it often to make sure I keep the final structure I want to use. This requiered me quite a few windows opened (~7) to keep track of all the pieces. I reallly should put everything in one file at first, setting up a complex space requires quite a bit of planning and a significant file creatiion/workspace organization to apply the way I want to do it.
```
Comment (helenemccarron) [inline styles/designs/generic/_splash-components.scss]: This is a file from a different branch and shouldn't be included here.

File: styles/designs/generic/_splash-spaces.scss [hunk 9]
```diff
@@ -0,0 +1,10 @@
+@import '_splash-components';
```
Comment (helenemccarron) [inline styles/designs/generic/_splash-spaces.scss]: same as above

File: styles/designs/generic/_titles-components.scss [hunk 10]
```diff
@@ -0,0 +1,38 @@
+// //TODO Figure out title structure
```
Comment (helenemccarron) [inline styles/designs/generic/_titles-components.scss]: same as above

File: styles/designs/generic/_titles-spaces.scss [hunk 11]
```diff
@@ -0,0 +1,27 @@
+//TOOD figure out title structure
```
Comment (helenemccarron) [inline styles/designs/generic/_titles-spaces.scss]: same as above

File: styles/designs/generic/_toc-components.scss [hunk 12]
```diff
@@ -0,0 +1,46 @@
+//TOC components
```
Comment (helenemccarron) [inline styles/designs/generic/_toc-components.scss]: same as above

File: styles/designs/generic/_toc-spaces.scss [hunk 13]
```diff
@@ -0,0 +1,27 @@
+@import 'toc-components';
```
Comment (helenemccarron) [inline styles/designs/generic/_toc-spaces.scss]: same as above

File: styles/templates/carnival/parts/_eoc-settings.scss [hunk 14]
```diff
@@ -1 +1,35 @@
-// EOC settings go here
+@import 'designs/carnival/_eoc-spaces.scss';
```
Comment (helenemccarron) [inline styles/templates/carnival/parts/_eoc-settings.scss]: I don't think this is part of this PR

File: styles/templates/carnival/parts/_figureimage-settings.scss [hunk 15]
```diff
@@ -0,0 +1,38 @@
+
+@include add_settings((
+    CaptionFont:(
```
Comment (helenemccarron) [inline styles/templates/carnival/parts/_figureimage-settings.scss]: As mentioned above, I would prefer to keep a container approach: define all caption items, titles items etc.

File: styles/templates/carnival/use.scss [hunk 16]
```diff
@@ -0,0 +1,5 @@
+//figureimage
```
Comment (helenemccarron) [inline styles/templates/carnival/use.scss]: did we decide to use a use file? cc @brittany-johnson

File: styles/templates/generic/_figure.scss [hunk 17]
```diff
@@ -0,0 +1,20 @@
+@include add_settings((
```
Comment (helenemccarron) [inline styles/templates/generic/_figure.scss]: I don't think it's necessary to update the generic template at this time

File: styles/templates/generic/parts/_eoc-settings.scss [hunk 18]
```diff
@@ -0,0 +1,41 @@
+@import './framework/_framework.scss';
```
Comment (helenemccarron) [inline styles/templates/generic/parts/_eoc-settings.scss]: i don't think the generic theme should be modified at this time

**Comment by helenemccarron** [comment]
Please make sure to update your branch it's out of date.

File: styles/books/chemistry/book.scss [hunk 19]
```diff
@@ -4,4 +4,11 @@
...
-//@use
\ No newline at end of file
+//@use
+@include use('CommonFigureStuff','common:::UnnumberedFigure');
+@include use('StandAloneImg','common:::AloneImg');
+
+@include use('CaptionFont','carnival:::CaptionFont');
+@include use('FigureNumberFont','carnival:::FigurenumFont');
+@include use('FigureTitle','carnival:::FiguretitleFont');
+@include use('PicPadding','carnival:::PaddingStuff');
```
Comment (tomjw64) [inline styles/books/chemistry/book.scss]:
```suggestion
@include use('PicPadding','carnival:::PaddingStuff');

```

File: styles/designs/common/pdf/_common-figure-components.scss [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+//unnumbered figure 
+$UnnumberedFigure:(
+    _name:"UnnumberedFigure",
+    _subselector: "",
+    _properties:(
+        display: enum('ValueSet:::NOT_NULL'),
+        margin-left: enum('ValueSet:::NOT_NULL'),
+        margin-right: enum('ValueSet:::NOT_NULL'),
+    )
+);
```
Comment (tomjw64) [inline styles/designs/common/pdf/_common-figure-components.scss]:
```suggestion
);

```

File: styles/designs/common/pdf/_common-figure-spaces.scss [hunk 21]
```diff
@@ -0,0 +1,13 @@
...
+    _components:($NumberedFigure)
+));
+
+@include create_superspace('common:::AloneImg',(
+    _components:($StandAloneImage)
+));
+
+@include create_superspace('common:::UnnumberedFigure',(
+    _components:($UnnumberedFigure)
+));
```
Comment (tomjw64) [inline styles/designs/common/pdf/_common-figure-spaces.scss]:
```suggestion
));

```

File: styles/designs/common/pdf/_common-figure-spaces.scss [hunk 22]
```diff
@@ -0,0 +1,13 @@
+@import '_common-figure-components';
```
Comment (tomjw64) [inline styles/designs/common/pdf/_common-figure-spaces.scss]:
```suggestion
@import 'common-figure-components';
```

File: styles/templates/generic/_colors.scss [hunk 23]
```diff
@@ -18,20 +17,14 @@
...
-    fontBodyColor: (_ref: "scheme:::bodyFontColor"),
     tocbg: (_ref: "scheme:::Grey1"),
     tocTitleBGColor: (_ref: "scheme:::B1"),
     tocTitleColor: (_ref: "scheme:::B2"),
     h1TitleColor: (_ref: "scheme:::O1"),
     h2TitleColor: (_ref: "scheme:::Y2"),
     h3TitleColor: (_ref: "scheme:::Y1"),
   )
-));
+));
```
Comment (tomjw64) [inline styles/templates/generic/_colors.scss]:
```suggestion
));

```

File: styles/templates/generic/_design.scss [hunk 24]
```diff
@@ -1,2 +1,3 @@
 @import 'designs/common/pdf/_common-spaces';
 
+@import 'designs/carnival/_figureimage-spaces.scss';
```
Comment (tomjw64) [inline styles/templates/generic/_design.scss]:
```suggestion
@import 'designs/carnival/figureimage-spaces';
```

File: styles/templates/generic/_typography.scss [hunk 25]
```diff
@@ -1,17 +1,15 @@
...
   typography: (
     sansFont: "IBM Plex Sans",
     sansFontLight: "IBM Plex Sans Light",
     sansFontMedium: "IBM Plex Sans Medium",
     baseFont: "IBM Plex Sans Regular",
     serifFont: "Muli Black",
-    baseLineHeight: 16px,
-    baseFontSize: 16px,
+    baseLineHeight: 9.5pt,
+    baseFontSize: 9.5pt,
```
Comment (tomjw64) [inline styles/templates/generic/_typography.scss]: Changing the generic template font size is outside the scope of this PR

File: styles/templates/template1/_design.scss [hunk 26]
```diff
@@ -1,2 +1,2 @@
-@import 'designs/common/pdf/_common-spaces';
-
+@import '../../designs/common/pdf/_common-figure-spaces.scss';
```
Comment (tomjw64) [inline styles/templates/template1/_design.scss]:
```suggestion
@import 'designs/common/pdf/common-figure-spaces';
```

File: styles/templates/template1/_design.scss [hunk 27]
```diff
@@ -1,2 +1,2 @@
...
+@import '../../designs/carnival/parts/_figureimage-spaces.scss';
```
Comment (tomjw64) [inline styles/templates/template1/_design.scss]:
```suggestion
@import 'designs/carnival/parts/figureimage-spaces';
```

File: styles/templates/template1/parts/_figureimage-settings.scss [hunk 28]
```diff
@@ -0,0 +1,38 @@
+
+@include add_settings((
+    CaptionFont:(
+        _selectors: ("[data-type='chapter'] > [data-type='page'] .os-figure > .os-caption-container > .os-caption"),
```
Comment (tomjw64) [inline styles/templates/template1/parts/_figureimage-settings.scss]: We should be nesting this in another component with a subselector rather than adding a large selector here.

File: styles/templates/template1/parts/_figureimage-settings.scss [hunk 29]
```diff
@@ -0,0 +1,38 @@
...
+        CaptionContainer:(
+            font-family:(_ref:'typography:::sansFont'), 
+            font-size: not_implemented(1.2rem), 
+        )
+    ),
+
+    FigureNumberFont:(
+        _selectors: ("[data-type = 'chapter'] > [data-type='page'] .os-figure> .os-caption-container > .os-number"),
```
Comment (tomjw64) [inline styles/templates/template1/parts/_figureimage-settings.scss]: We should be nesting this in another component with a subselector rather than adding a large selector here.

File: styles/templates/template1/parts/_figureimage-settings.scss [hunk 30]
```diff
@@ -0,0 +1,38 @@
...
+        FigureNumContainer:(
+            font-family: (_ref:'typography:::serifFont'),
+            font-weight: 800,
+            font-size: not_implemented(1.2rem),
+            color: (_ref:"colorMap:::figureNumberColor"),
+        )
+    ),
+
+    FigureTitle:(
+        _selectors: ("[data-type = 'chapter'] > [data-type='page'] .os-figure > .os-caption-container > .os-title-label"),
```
Comment (tomjw64) [inline styles/templates/template1/parts/_figureimage-settings.scss]: We should be nesting this in another component with a subselector rather than adding a large selector here.

File: styles/templates/generic/_colors.scss [hunk 31]
```diff
@@ -1,6 +1,5 @@
 //POSSIBLE BREAKDOWN: "raw" settings used with _ref to call globally used settings?
-//Define all hex values first, then create a map referencing the values belwo
-//TODO rename those colors variables to something a tad more descriptive
```
Comment (helenemccarron) [inline styles/templates/generic/_colors.scss]: can you please restore this comment (maybe minus the typo)

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 32]
```diff
@@ -0,0 +1,17 @@
+@import '_figureimage-components';
+
+@include create_superspace('carnival:::CaptionFont',(
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: Don't think we should be naming based on CSS properties

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 33]
```diff
@@ -0,0 +1,17 @@
...
+    _components: ($Caption_Container)
+));
+
+@include create_superspace('carnival:::FigurenumFont',(
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: Same as above

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 34]
```diff
@@ -0,0 +1,17 @@
...
+    _components: ($FigureNum_Container)
+));
+
+@include create_superspace('carnival:::FiguretitleFont',(
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: Same as above

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 35]
```diff
@@ -0,0 +1,17 @@
...
+    _components: ($Figure_Container)
+));
+
+@include create_superspace('carnival:::PaddingStuff',(
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: Same as above. Also should avoid naming things `Stuff`

File: styles/designs/common/pdf/_common-figure-components.scss [hunk 36]
```diff
@@ -0,0 +1,30 @@
+//numbered <figure> 
+$NumberedFigure:(
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-components.scss]: Because there is no `_subselector`, this should be named `NumberedFigureContainer`

File: styles/designs/common/pdf/_common-figure-components.scss [hunk 37]
```diff
@@ -0,0 +1,30 @@
...
+    _properties:(
+        display: enum('ValueSet:::NOT_NULL'),
+        margin-left: enum('ValueSet:::NOT_NULL'),
+        margin-right: enum('ValueSet:::NOT_NULL'),
+    )
+);
+
+//images w/o caption/figure number
+$StandAloneImage:(
+    _name: "AloneImg",
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-components.scss]: Maybe we could name this something like `FigureNoCaption` or `NoCaptionFigure`

File: styles/designs/common/pdf/_common-figure-components.scss [hunk 38]
```diff
@@ -0,0 +1,30 @@
...
+    _subselector: "",
+     _properties:(
+         float:enum('ValueSet:::NOT_NULL'),
+     )
+);
+
+//unnumbered figure 
+$UnnumberedFigure:(
+    _name:"UnnumberedFigure",
+    _subselector: "",
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-components.scss]: no `_subselector` so it should contain the name Container

File: styles/designs/common/pdf/_common-figure-spaces.scss [hunk 39]
```diff
@@ -0,0 +1,13 @@
+@import 'common-figure-components';
+
+@include create_superspace('common:::FigureStuff',(
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-spaces.scss]: Please rename to remove `Stuff`

File: styles/designs/common/pdf/_common-figure-spaces.scss [hunk 40]
```diff
@@ -0,0 +1,13 @@
...
+    _components:($NumberedFigure)
+));
+
+@include create_superspace('common:::AloneImg',(
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-spaces.scss]: `Img` -> `Figure`

File: styles/templates/generic/_colors.scss [hunk 41]
```diff
@@ -18,15 +19,9 @@
...
     Grey1: #EEEAE1,
-    bodyFontColor: #333333
   ),
   //uses the color scheme above. This is used to define all colors in one location where the values can be reassigned easily for each new template. Maybe find a way to make this file reusable?
-
-  //usage in settings file:
-  //color: (_ref: "colorMap:::h1TitleColor")
-
   colorMap: (
-    fontBodyColor: (_ref: "scheme:::bodyFontColor"),
```
Comment (brittany-johnson) [inline styles/templates/generic/_colors.scss]: Just wondering why this is being removed, seems like it would be needed?

File: styles/templates/template1/parts/_common-figure-settings.scss [hunk 42]
```diff
@@ -0,0 +1,17 @@
+@include add_settings((
+    CommonFigureStuff:(
```
Comment (brittany-johnson) [inline styles/templates/template1/parts/_common-figure-settings.scss]: Rename to remove `Suff`

File: styles/templates/template1/parts/_common-figure-settings.scss [hunk 43]
```diff
@@ -0,0 +1,17 @@
...
+        _selectors: ("[data-type='chapter'] > [data-type='page'] .os-figure > figure"),
+        UnnumberedFigure:(
+            display: block,
+            margin-left: auto,
+            margin-right: auto,
+        )
+    ),
+
+    StandAloneImg:(
+        _selectors: (":not(figure) > :not(figure) > :not(figure) > img"),
```
Comment (brittany-johnson) [inline styles/templates/template1/parts/_common-figure-settings.scss]: Is this selector working as intended? It doesn't look like it would work from a glance.

File: styles/books/chemistry/book.scss [hunk 44]
```diff
@@ -1,13 +1,14 @@
...
-@import "templates/carnival/_colors.scss";
-@import 'templates/carnival/_typography.scss';
-@import 'templates/carnival/parts/_eoc-settings.scss';
+//book specific styles
 
+//@use
+@include use('CommonFigureStuff','common:::UnnumberedFigure');
+@include use('StandAloneImg','common:::AloneImg');
 
-@include use('KeyTerm','eocKeyTerm');
```
[the remark this answers is not in the document]
Comment (richelleyy) [reply styles/books/chemistry/book.scss]: @brittany-johnson Yes. I thought I just need what I have for my current PR?

**Comment by helenemccarron** [comment]
It feels like this branch isn't up to date or in sync with m.m. You shouldn't have to delete the files from the PR itself, those files should not be present if they were deleted as part of another PR. There seems to be quite a few conflicts, I would work on making sure that only the changes that are pertinent to this PR are included.

File: styles/templates/template1/parts/_common-figure-settings.scss (hunk 43 above)
Comment (brittany-johnson) [reply styles/templates/template1/parts/_common-figure-settings.scss]: @richelleyy

Comment (richelleyy) [reply styles/templates/template1/parts/_common-figure-settings.scss]: @brittany-johnson yes, it works. Thomas and I figured out it might be the only way I can select the images I want

File: styles/templates/generic/_colors.scss (hunk 41 above)
Comment (richelleyy) [reply styles/templates/generic/_colors.scss]: I didn't use this color file, so not sure why it's removed?

Comment (helenemccarron) [reply styles/templates/generic/_colors.scss]: I added that color as a placeholder, especially the value, but we should define it, rather than delete it, we should make that the default value is enabled

File: styles/templates/generic/_typography.scss (hunk 25 above)
Comment (helenemccarron) [reply styles/templates/generic/_typography.scss]: Please undo this change, it has been addressed in another PR.

File: styles/books/chemistry/book.scss [hunk 45]
```diff
@@ -2,10 +2,18 @@
 @import '../../templates/template1/design.scss';
 @import '../../templates/template1/settings.scss';
 
-@include use('RootTypography', "common:::RootTypography");
```
Comment (tomjw64) [inline styles/books/chemistry/book.scss]: This should not have been removed.

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 46]
```diff
@@ -0,0 +1,17 @@
...
+
+@include create_superspace('carnival:::Caption',(
+    _components: ($Caption)
+));
+
+@include create_superspace('carnival:::FigureNum',(
+    _components: ($FigureNum)
+));
+
+@include create_superspace('carnival:::FigureTitle',(
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: Many of these components need to be nested within a greater `FigureContainer` type element. With this model, we are going to have to unnecessarily repeat many selectors. Your recent PRs have made better use of component nesting. Use those for reference.

File: styles/templates/generic/_typography.scss [hunk 47]
```diff
@@ -11,7 +9,7 @@
     baseFont: "IBM Plex Sans Regular",
     serifFont: "Muli Black",
     baseLineHeight: 16px,
-    baseFontSize: 16px,
+    baseFontSize: 16px,
```
Comment (tomjw64) [inline styles/templates/generic/_typography.scss]: This formatting change is undesired.

File: styles/templates/template1/parts/_common-figure-settings.scss (hunk 43 above)
Comment (tomjw64) [reply styles/templates/template1/parts/_common-figure-settings.scss]: @brittany-johnson Yup, we have to do this in recipes too :( CSS isn't all roses.

File: styles/templates/generic/_typography.scss [hunk 48]
```diff
@@ -1,8 +1,6 @@
 // @import "compass/typography" or some other vertical rhythm functions
 
-//usage in settings file: font-family: (_ref: 'typography:::sansFont'),
```
Comment (helenemccarron) [inline styles/templates/generic/_typography.scss]: please discard all changes in the generic template and move them to the template1 instead. This file should not be modified

File: styles/books/chemistry/book.scss [hunk 49]
```diff
@@ -3,9 +3,18 @@
 @import '../../templates/template1/settings.scss';
 
 @include use('RootTypography', "common:::RootTypography");
-
+//book specific styles
 
 //@use
+@include use('CommonFigure','common:::CommonFigure');
```
Comment (helenemccarron) [inline styles/books/chemistry/book.scss]: Should these settings be renamed to just figure? The pattern for other figure and image elements doesn't use "common" in the settings name.
`use('CommonFigure','common:::CommonFigure');`
vs
`use('CaptionFont','carnival:::Caption');`

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 50]
```diff
@@ -0,0 +1,45 @@
...
+    _subselector: "",
+     _properties:(
+    )
+);
+
+$Caption:(
+    _name: "Caption",
+    _subselector: "> .os-caption-container > .os-caption",
+     _properties:(
+         font-family: enum('ValueSet:::NOT_NULL'),
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_figureimage-components.scss]: In order to merge into n.n, you will have to rename your `ValueSet`

File: styles/designs/carnival/parts/_figureimage-spaces.scss [hunk 51]
```diff
@@ -0,0 +1,14 @@
+@import '_figureimage-components';
+
+@include create_superspace('carnival:::Figure',(
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_figureimage-spaces.scss]: In order to merge your PR you will have to rename the spaces to shapes.

File: styles/books/chemistry/book.scss (hunk 49 above)
Comment (richelleyy) [reply styles/books/chemistry/book.scss]: @helenemccarron I named it like this because there're other figure elements `NoCaptionFigure` and `NumberedFigure`, `CommonFigure` refers to anything else.

File: styles/designs/carnival/parts/_figureimage-components.scss (hunk 50 above)
Comment (helenemccarron) [reply styles/designs/carnival/parts/_figureimage-components.scss]: Also, do not forget to point your branch at n.n and update it with Brittany's changes :D

File: styles/books/chemistry/book.scss [hunk 52]
```diff
@@ -3,9 +3,18 @@
...
+@include use('NoCaptionFigure','common:::NoCaptionFigure');
+
+@include use('CaptionFont','carnival:::Caption');
+@include use('FigureNumberFont','carnival:::FigureNum');
+@include use('FigureTitle','carnival:::FigureTitle');
+@include use('PicCaption','carnival:::FigureCaption');
```
Comment (tomjw64) [inline styles/books/chemistry/book.scss]: It looks like some of these superspaces have since been removed. It seems like this should yield an error. Is that not the case?

Comment (richelleyy) [reply styles/books/chemistry/book.scss]: @tomjw64 it's not yielding errors but I changed it.

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 53]
```diff
@@ -0,0 +1,45 @@
...
+         font-weight: enum('ValueSet:::NULLABLE'),
+         color: enum('ValueSet:::NULLABLE'), 
+         font-size: enum('ValueSet:::NOT_NULL'),
+    )
+);
+
+$FigureCaption:(
+    _name: "PicCaption",
+    _subselector: "> figure > figure",
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_figureimage-components.scss]: How is this targeting the caption? It seems to be targeting figures within figures?

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 54]
```diff
@@ -0,0 +1,45 @@
+$FigureContainer:(
+    _name: "FigureContainer",
+    _subselector: "",
+     _properties:(
+    )
+);
+
+$Caption:(
+    _name: "Caption",
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-components.scss]: What's the difference between `$Caption` and `$FigureCaption` ?

File: styles/designs/carnival/parts/_figureimage-shapes.scss [hunk 55]
```diff
@@ -0,0 +1,14 @@
+@import '_figureimage-components';
+
+@include create_superspace('carnival:::Figure',(
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_figureimage-shapes.scss]: If your branch is up to date with `n.n`,  `create_superspace` should not work because it was replaced with `create_shape`

File: styles/designs/common/pdf/_common-figure-shapes.scss [hunk 56]
```diff
@@ -0,0 +1,13 @@
+@import 'common-figure-components';
+
+@include create_superspace('common:::NumberedFigure',(
```
Comment (brittany-johnson) [inline styles/designs/common/pdf/_common-figure-shapes.scss]: Same comment as above, `create_superspace` needs to be updated

File: styles/templates/generic/_typography.scss [hunk 57]
```diff
@@ -1,8 +1,6 @@
 // @import "compass/typography" or some other vertical rhythm functions
 
 //usage in settings file: font-family: (_ref: 'typography:::sansFont'),
-
-/* stylelint-disable-next-line meowtec/no-px */
```
Comment (helenemccarron) [inline styles/templates/generic/_typography.scss]: Why was this deleted? This is a functional comment, it's needed for the PR to pass.
